etcdserver/server.go

```diff
@@ -527,6 +539,32 @@ func NewServer(cfg ServerConfig) (srv *EtcdServer, err error) {
 	}
 	srv.r.transport = tr
 
+	srv.goAttach(func() {
```

Comment:
why this needs to be async? can we start the network first? then wait here? then decide to advance ticks or not. then start the raft routine?

Reply:
also if it works like what i described, we do not need the lock to protect the tick.

Reply:
It needs to be async, since we start peer handler in embed package, after we create etcd server here.

Addressed others. PTAL.